api/metadata.go

```diff
@@ -103,3 +99,47 @@ func (api *CantabularMetadataExtractorAPI) GetMetadataDataset(ctx context.Contex
 	return md, err
 
 }
+
+// OverrideMetadataTable modifies the dimensions and results of the MetadataTableQuery
+// to always use "ltla".  This is the geocode used in the recipe and we need to ensure
+// the result from the metadata server matches the recipe.  This ensures also the
+// following GetMetadataDataset uses "ltla".
+func OverrideMetadataTable(dims []string, mt *cantabular.MetadataTableQuery) error {
+	found := 0
+	for i, v := range dims {
+		if inSlice(v, validGeo) {
+			dims[i] = geoCodeOverride
+			found++
+		}
+	}
+
+	if found != 1 {
+		return fmt.Errorf("dimensions : %w", errNotOneGeocode)
+	}
```

Comment:
We could stop iterating as soon as we found more than 1?

Reply:
I think its a micro optimization which isn't worth it. The slices are small only 3 or 4 tops.